usdigitalresponse/usdr-gost
# [DRAFT] 266 hide links on dashboard 2

### Ticket #
#266
### Description
See ticket
### Screenshots / Demo Video
n/a
### Testing

To manually test (using local database):
- Get clean local database: `docker compose exec app yarn db:seed`
- Verify dashboard links are visible
- Comment out as shown in the following diff:

```
--- a/packages/server/seeds/dev/ref/grants.js
+++ b/packages/server/seeds/dev/ref/grants.js
@@ -754,7 +754,7 @@ const grantsInterested = [
         updated_at: daysAhead(5),
         interested_code_id: interestedId('Will Support'),
     },
-    {
+/*    {
         grant_id: grants[6].grant_id,
         agency_id: usdr,
         user_id: 1,
@@ -762,6 +762,7 @@ const grantsInterested = [
         updated_at: daysAhead(6),
         interested_code_id: interestedId('Will Support'),
     },
+*/
```

- Re-run `docker compose exec app yarn db:seed`
- Verify dashboard links are gone

### Checklist
